main/ertp/api/issuer.md

```diff
@@ -1,214 +1,263 @@
+Split a single `payment` into two new `payments`, A and B, according to `paymentAmountA`. 
+For example, if the `payment` is for 10 Quatloos, and `paymentAmountA` is 3 Quatloos,
+it returns an array of two `payments` with balances of 3 Quatloos and 7 Quatloos.
 
-```js
-const { mint, issuer, amountMath } = makeIssuerKit('bucks');
-const oldPayment = mint.mintPayment(amountMath.make(20));
+The original `payment` is burned. If the original `payment`
+is a `promise`, the operation proceeds after the `promise` resolves.
 
-const [paymentA, paymentB] = issuer.split(oldPayment, amountMath.make(10));
+```js
+const { mint: quatloosMint, issuer: quatloosIssuer, amountMath: quatloosAmountMath } = makeIssuerKit('quatloos');
+const oldPayment = quatloosMint.mintPayment(quatloosAmountMath.make(20));
+// After the split, paymentA has 5 quatloos and paymentB has 15.
+const [paymentA, paymentB] = quatloosIssuer.split(oldPayment, quatloos.AmountMath.make(5));
 ```
 
 ## issuer.splitMany(payment, amountArray)
 - `payment` `{Payment}`
 - `amountArray` `{Array <Amount>}`
 - Returns: `{Array <Payment>}`
 
-Split a single payment into multiple payments. The resulting array of payments will be as long as `amountArray`, and the payments will have amounts
-corresponding to `amountArray`. If the original `payment` is a promise, the operation will proceed after the promise resolves.  If the amounts in
-`amountArray` don't add up to the value of `payment`, the operation fails. The original `payment` will be burned.
+Split a single `payment` into multiple `payments`. The resulting array of `payments` is
+as long as `amountArray`, and the `payments` will have `amounts` corresponding to 
+the `amountArray` contents. The original `payment` is burned. If the original `payment` 
+is a `promise`, the operation proceeds after the `promise` resolves.  If the `amounts` 
+in `amountArray` don't add up to the value of `payment`, the operation fails. The `brands`
+of the `amountArray` `amounts` must all be the same as the `payment` `brand`.
 
 ```js
-const { mint, issuer, amountMath } = makeIssuerKit('fungible');
-const oldPayment = mint.mintPayment(amountMath.make(100));
-const goodAmounts = Array(10).fill(amountMath.make(10));
-
-const arrayOfNewPayments = issuer.splitMany(oldPayment, goodAmounts);
-```
+const { mint: quatloosMint, issuer: quatloosIssuer, amountMath: quatloosAmountMath} = makeIssuerKit('quatloos');
+const oldPayment = quatloosMint.mintPayment(quatloos.AmountMath.make(100));
+const goodAmounts = Array(10).fill(quatloosAmountMath.make(10));
 
-Note that the total amount in the `amountArray` must equal the amount in the original `payment`:
+const arrayOfNewPayments = quatloos.Issuer.splitMany(oldPayment, goodAmounts);
 
-```js
-const { mint, issuer, amountMath } = makeIssuerKit('fungible');
-const payment = mint.mintPayment(amountMath.make(1000));
+// The total amount in the amountArray must equal the original payment amount
```

Comment:
This comment seems slightly misplaced. Should it go a couple of lines below? Or should we change it?